cmake/setup_compiler_flags_msvc.cmake

```diff
@@ -27,6 +27,14 @@
 #                      #
 ########################
 
+IF(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "19.10" )
+  MESSAGE(FATAL_ERROR "\n"
+    "You're using an old version of the Visual Studio C++ Compiler!\n"
+    "You need at least version 14.1 (VS 2017).\n"
```

Comment:
I have never compiled deal.II on Windows, but the check says `19.10` whereas your comment says `14.1`. Is this a typo or can you expand?

Reply:
yeah, sorry. I added a comment (and it does work correctly as is)